src/ast/data_type.rs

```diff
@@ -297,3 +293,34 @@ impl fmt::Display for TimezoneInfo {
         }
     }
 }
+
+/// Additional information for `NUMERIC`, `DECIMAL`, and `DEC` data types
+/// following the 2016 [standard].
+///
+/// [standard]: https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#datetime-type
```

Comment:
this link seems incorrect (as it points to datetime-type)?

Reply:
Fixed, thanks!